packaging/generic-deb/ecs.service

```diff
@@ -0,0 +1,33 @@
+# Copyright 2014-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
In some other comments I suggested moving this file around during packaging, but instead you should probably just move it under the `debian/` subdirectory in git.

Reply:
done